rollup.build.js

```diff
@@ -0,0 +1,39 @@
+import babel from 'rollup-plugin-babel'
+
+export default {
+  input: 'lib/indefinite.js',
+  output: [
```

Comment:
Nice, thanks for taking this little extra bit on (I mean, it looks like it was pretty straight forward, but . . . .still not nothing).

Reply:
I don't know how rollup works . . . will it automatically create minified versions of each of these?

Reply:
Also, we'll need to update the README to at least mention support for various formats.